reflect.go

```diff
@@ -680,6 +682,25 @@ func (t *Schema) genericKeywords(tags []string, parent *Schema, propertyName str
 	}
 }
 
+// read struct tags for boolean type keyworks
+func (t *Schema) booleanTags(tags []string) {
```

Comment:
To be consistent, I'd suggest calling this `booleanKeywords`.

Reply:
Sure thing!